### ISSUE TITLE
 error processing GARD job

### ISSUE THREAD

**Issue by geofrunz** [veg/hyphy#726]
Dear all, 
recently I've  not been able to obtain any results from GARD analysis on datamonkey. 
After each attempt I get the following message "We're sorry, there was an error processing your job."
I've tried with different dataset, including the Influenza A H5N1 provided as example in datamonkey...
 Maybe I'm missing something, but I'm using the default parameters..
Many thanks for any advice you could provide me
geofrunz

**Comment by stevenweaver** [comment]
Dear @geofrunz, 

Thank you for bringing this issue to our attention. The issue should be resolved and reflected on datamonkey.org now. 

Best,
Steven

**Comment by kmgibson** [comment]
Dear @stevenweaver and @spond,

I am experiencing this same issue currently. I am submitting a dataset with 59 sequences with 99 sites. The Job Queue says my job is complete, but I cannot get any results on the page where I submitted the dataset - exactly like geofrunz's problem with the same error message.

I have retried running this dataset a few times, as well as redownloading the dataset and trying to run it again. Same problem each time. I am using the default parameters as well.

Thanks in advance for any assistance.

Best,
Keylie
